### ISSUE TITLE
Unable to load narrator natives

### ISSUE THREAD

**Issue by JamiesWhiteShirt** [FabricMC/fabric-loom#81]
This is non-critical as the game runs fine without them. The narrator is simply unavailable as a result.

The narrator uses JNA to load the natives, and the natives cannot be loaded by [the available mechanisms](https://github.com/java-native-access/jna/blob/master/www/GettingStarted.md) in my Windows development environment. It appears the vanilla launcher creates a temporary directory and sets the `jna.library.path` system property. Perhaps loom should extract natives similarly?

```
[Narrator/INFO]: [STDERR]: java.lang.UnsatisfiedLinkError: com.mojang.text2speech.NarratorWindows$SAPIWrapperSolutionDLL.queue(JLcom/sun/jna/WString;Z)V
Exception in thread "Narrator" [22:03:11] [Narrator/INFO]: [STDERR]: 	at com.mojang.text2speech.NarratorWindows$SAPIWrapperSolutionDLL.queue(Native Method)
[Narrator/INFO]: [STDERR]: 	at com.mojang.text2speech.NarratorWindows$Message.apply(NarratorWindows.java:110)
[Narrator/INFO]: [STDERR]: 	at com.mojang.text2speech.NarratorWindows$NarratorThread.run(NarratorWindows.java:80)
[Narrator/INFO]: [STDERR]: 	at java.base/java.lang.Thread.run(Thread.java:835)
```

**Comment by Chocohead** [comment]
This is just a Windows thing right? Just as there's no point extracting for Unix if it doesn't need it

**Comment by modmuss50** [comment]
I think we removed the native extraction from loom when lwjgl no longer required it, but then the narrator came along.

**Comment by asiekierka** [comment]
@modmuss50 want to take a look at it?

**Comment by modmuss50** [comment]
Should be fixed in loom 0.2.6, you may need to regnerate your run configs.
